client/src/pages/RentalProcess.js

```diff
@@ -0,0 +1,127 @@
+import React from "react";
+import styled from "@emotion/styled";
+
+const Form = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const AdressLine1 = styled.div`
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  div {
+    display: flex;
+    flex-direction: column;
+  }
+  & :first-child {
+    width: 70%;
+  }
+  & :last-child {
+    width: 20%;
+  }
+`;
+
+const AdressLine2 = styled.div`
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  div {
+    display: flex;
+    flex-direction: column;
+  }
+  & :first-child {
+    width: 35%;
+  }
+  & :last-child {
+    width: 55%;
+  }
+`;
```

Comment:
The address lines are very similar.
You could reuse the styling if you style an existing component or use composition:
https://emotion.sh/docs/styled#styling-any-component
https://emotion.sh/docs/composition

e.g.
```jsx
const AddressLine = styled.div`
  display: flex;
  justify-content: space-between;
  width: 100%;
  div {
    display: flex;
    flex-direction: column;
  }
`;

const AddressLine1 = styled(AddressLine)`
   & :first-child {
    width: 35%;
  }
  & :last-child {
    width: 55%;
  }
`;

const AddressLine2 = styled(AddressLine)`
   & :first-child {
    width: 45%;
  }
  & :last-child {
    width: 45%;
  }
`;
```

In general, `AddressLine1` is not a good name. What is special about this component or styling? Maybe you find a better name. Sometimes it's helpful to take a look at existing UI frameworks like Bootstrap or Material UI.